### PR TITLE
Refactoring + Adding info on data organization

### PR DESCRIPTION
**Pull request by EmmaRenauld**
As discussed with @arnaudbore

### CODE REVIEW

File: source/standards/data_organization_in_our_lab.rst [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+- Please keep all folders names in lowercase (this is true when coding, but also generally everywhere, like in BrainData for instance), and prefer the use of underscores. Ex: my_project instead of MyProject.
+
+- If you want to start a project with an existing dataset: You should find the raw data you are looking for in BrainData/databases. It is also possible that other lab member have alread processed this data exactly the same way as you want. Please check in BrainData/processedData. If you can't find your data, find the slack channel #help_me_find_data.
+
+- If you want to save a finished project with...
+
+    - an existing database in our lab: You can go in BrainData/processedData and create a folder **with a significative name**. Please add information on how your data was processed. Ask Arnaud if you have questions or if you think your data should rather be kept on Beluga.
+
+    - a new database: Please talk with Arnaud if you have aquired data from a new database. Depending on the source/format/characteristic of the data, we will probably add the raw data to BrainData/databases.
```
Comment (arnaudbore) [inline source/standards/data_organization_in_our_lab.rst:16]: acquired

File: source/standards/data_organization_in_our_lab.rst [hunk 2]
```diff
@@ -0,0 +1,16 @@
+.. _ref_organization_standards:
```
Comment (arnaudbore) [inline source/standards/data_organization_in_our_lab.rst:1]: Additional information.

Start a project using raw data:
-> existing data check databases
-> data from open databases -> coordinate with arnaud
-> non existing data (meaning you acquire data)-> coordinate with arnaud

Start project using already preproc data
-> databases/your_database/derivatives
-> processedData/your_database/

Once you have run a pipeline coordinate with arnaud to put it in databases/your_database/derivative

File: source/arriving/welcome.rst [hunk 3]
```diff
@@ -3,6 +3,17 @@
 Onboarding new students
 =======================
 
+Where to start?
+""""""""""""""""
+
+1. When you first arrive in the lab, take the time to set everything well on your computer. See below for a list of useful accounts you should create and software/tools you should install.
+
+2. You can check each of our tools pages for a description and for some tutorials (Tractoflow, Recobundles, etc.).
```
Comment (arnaudbore) [inline source/arriving/welcome.rst:11]: each tool's page

File: source/standards/data_organization_in_our_lab.rst (hunk 2 above)
Comment (arnaudbore) [reply source/standards/data_organization_in_our_lab.rst:1]: DO NOT duplicate data

File: source/our_tools/tractoflow.rst [hunk 4]
```diff
@@ -207,3 +207,8 @@ Using config files
...
 
 The tractoflow command can also be ran with most options listed in a config file such as `this one <https://github.com/scilus/tractoflow/blob/master/nextflow.config>`_. You simply have to keep the nextflow.config file in the directory from where you run your command. However, a better use of the config files would be to use them for consultation only and to give all the parameters explicitly, manually, when launching tractoflow. This may avoid confusion on the default parameters. Be careful, thus, not to keep the config file in your directory if you are not sure how to use them!
+
+After tractoflow
+*****************
+
+Once the preprocessing and the tracking are done, scilpy also contains scripts to separate your data into known bundles (Recobundles), vizualize your results (ex, scil_visualize_bundles_mosaic.py), compute statistics on your bundles and so on. One particularly important tool is the **Quality Check**. You should strongly consider verifying the quality of your tractoflow process with our QC tool (:ref:`ref_QC`). Once this step is done, talk with Arnaud to find the best way to save your data and share it with the lab.
```
Comment (arnaudbore) [inline source/our_tools/tractoflow.rst:214]: I did not read everything but when working on compute canada do not download the singularity. Check rrg-descotea/singularities

Comment (EmmaRenauld) [reply source/our_tools/tractoflow.rst:214]: I just checked. For me, this folder is empty?

Comment (arnaudbore) [reply source/our_tools/tractoflow.rst:214]: Yes it's empty but should not be in the future. :+1:

File: source/standards/data_organization_in_our_lab.rst (hunk 2 above)
Comment (arnaudbore) [reply source/standards/data_organization_in_our_lab.rst:1]: I think my way of presenting the different options is smoother and you are still missing some information here.

**Pull request closed without merging**